### ISSUE TITLE
mesa-17.3.1-1

### ISSUE THREAD

**Issue by udeved** [artix-linux/world#51]
Our mesa package has a problem with 3d acceleration.

If I try to start eg 0ad with new mesa, it won't find glx.
I suspect this is caused by recent drm path changes in mesa.
Reverting to mesa-17.3.0 works.

**Comment by thefallenrat** [comment]
Sorry if this wasn't related, but I think this was caused by `xorg-server-1.19.6-2` was built against `mesa-17.3.1-1` instead of `mesa-17.3.1-2` like I was intended to. Evidenced in 1395eff build log : 

```
resolving dependencies...
looking for conflicting packages...
warning: dependency cycle detected:
warning: libglvnd will be installed before its mesa dependency
warning: dependency cycle detected:
warning: harfbuzz will be installed before its freetype2 dependency

Package (87)               New Version          Net Change  Download Size
...
world-testing/mesa         17.3.1-1              55.60 MiB
...
```

Let's see if rebuilding the `xorg-server` fix the issue (4403adc)

**Comment by udeved** [comment]
Yeah, I suspected something xorg.
The xorg itself has 3d in plasma, but programs using glx don't work.
